lib/oidc/getUserInfo.ts

```diff
@@ -36,11 +36,23 @@ export async function getUserInfo<T extends CustomUserClaims = CustomUserClaims>
     return Promise.reject(new AuthSdkError('getUserInfo requires an ID token object'));
   }
 
-  return httpRequest(sdk, {
+  const options: any = {
     url: accessTokenObject.userinfoUrl,
     method: 'GET',
     accessToken: accessTokenObject.accessToken
-  })
+  };
+
+  if (sdk.options.dpop) {
+    const keyPair = await findKeyPair(accessTokenObject.dpopPairId);
+    const proof = await generateDPoPProof({ ...options, keyPair });
+    options.headers = {
```

Comment:
Reuse `getDPoPAuthorizationHeaders` util?

Reply:
Agreed, is there any reason why the util can't be reused here?